packages/genesys-spark-components/src/components/legacy/gux-modal-legacy/example.html

```diff
@@ -1,4 +1,26 @@
 <h1>gux-modal-legacy</h1>
+
+<div>
```

Comment:
Perhaps there's a way to make the legacy component example migration warning reusable, like having a separate component you can reference in each of the legacy component example files. Maybe it could save time for future releases.